explainability-service/src/main/java/org/kie/trustyai/service/payloads/values/reconcilable/serializers/ReconcilableFieldSerializer.java

```diff
@@ -0,0 +1,37 @@
+package org.kie.trustyai.service.payloads.values.reconcilable.serializers;
+
+import java.io.IOException;
+
+import org.kie.trustyai.service.payloads.values.DataType;
+import org.kie.trustyai.service.payloads.values.reconcilable.ReconcilableField;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+// this serializers ReconcilableFeatures/Outputs in a way that minimally modifies the existing documented JSON schema"
```

Comment:
Do we have any test that check the output JSON?

Reply:
I'll create a ticket https://github.com/trustyai-explainability/trustyai-explainability/issues/298